### PR TITLE
Run frontend on IONOS

### PR DESCRIPTION
**Pull request by jennchenn**
## Notion ticket link

[Run frontend on IONOS](https://www.notion.so/uwblueprintexecs/Run-frontend-on-IONOS-784cbebb1d4a4b5f92999315e8056c38)

## Implementation description

- Set up frontend to display at [https://www.planetread.org/aml_platform](https://www.planetread.org/aml_platform) *it's running an older vers of main
- NOTE before merging+tagging: will have to re-init git ftp because I pushed to it while testing this branch

## Steps to test

1. Poke around [https://www.planetread.org/aml_platform](https://www.planetread.org/aml_platform) and make sure things look okay
2. Poke around locally and make sure things look okay
3. Poke around on firebase preview and make sure things look okay

## What should reviewers focus on?

- Do the routes all work properly?

## Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] For backend changes, I have run the appropriate linters: `docker exec -it planet-read_py-backend_1 /bin/bash -c "black . && isort --profile black ."` and I have generated new migrations: `flask db migrate -m "<your message>"`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/userProfile/AssignedStoryTranslationsTable.tsx [hunk 1]
```diff
@@ -280,12 +280,12 @@ const AssignedStoryTranslationsTable = ({
   const generateStoryTranslationLink = (translation: StoryTranslation) => {
     const suffix = `${translation.storyId}/${translation.storyTranslationId}`;
     if (isAdmin) {
-      return `/story/${suffix}`;
+      return `#/story/${suffix}`;
```
Comment (jennchenn) [inline frontend/src/components/userProfile/AssignedStoryTranslationsTable.tsx:283]: Is this link supposed to be `story/storyId/storyTranslationId`? I seem to be getting a 404 here but not sure what changed

**Review by gaoxk — COMMENTED** [review]
All the links work well! Frontend is functioning as expected except for 2 things: 
image
logging in with google auth is broken

image

resetting password does not appear to work
The commonality between these two issues is that they involve 3rd parties (firebase, googleauth) so I wonder if it's a permissions/firewall problem. What's puzzling is that authenticating users relies entirely on firebase, and we clearly have no issues logging in. I wonder what specifically is problematic about firebase reset password. 
I am perfectly happy to push this to a separate fix ticket since it could require more research. What do you think? Got any hunches?

And finally a confirming question: is the frontend on the platform is always the most recent release?

**Comment by jennchenn** [comment]
> All the links work well! Frontend is functioning as expected except for 2 things: image logging in with google auth is broken
> 
> image
> 
> resetting password does not appear to work The commonality between these two issues is that they involve 3rd parties (firebase, googleauth) so I wonder if it's a permissions/firewall problem. What's puzzling is that authenticating users relies entirely on firebase, and we clearly have no issues logging in. I wonder what specifically is problematic about firebase reset password. I am perfectly happy to push this to a separate fix ticket since it could require more research. What do you think? Got any hunches?
> 
> And finally a confirming question: is the frontend on the platform is always the most recent release?

I think the `redirect_uri_mismatch` issue requires us to add the planetread domain to our client ID in the Google console (see [here](https://stackoverflow.com/questions/11485271/google-oauth-2-authorization-error-redirect-uri-mismatch))--I've done this now I believe, but not sure how long it takes for the changes to take effect / if another fix is needed. 

The reset password issue I think was a problem with not including the necessary env variables in the heroku deployment. If you try now I think it should work.

Re releases: yes that is correct going forward. The deployed version right now is just whatever version of main I most recently branched off of. Because I keep pushing to the server while developing, there is some weirdness with the git commit history that I need to fix before I merge my branch into main, so I didn't push the latest ver. of this branch up to the server yet. I can definitely do that, but just assumed we would be doing a new release soon after this PR is merged anyway, so wanted to save the trouble of having to reset the git ftp commit history twice.

**Review by gaoxk — APPROVED** [review]
reset password looks good now. Still getting the redirect issue with google auth so I can log a separate ticket for that. 

Great work!! Thank you for pushing through all the challenges and uncertainty 🙌 Can you generate a new release once this is merged in?

**Pull request merged**